backend/tests/conftest.py

```diff
@@ -228,10 +228,23 @@ def cleanup():
     }
 
 
+@pytest.fixture(scope="module")
+def rabbitmq(rabbitmq_container: dict[int, int] | None, reload_settings_before_each_module) -> dict[int, int] | None:
+    if (
+        rabbitmq_container
+        and INFRAHUB_USE_TEST_CONTAINERS
```

Comment:
If `INFRAHUB_USE_TEST_CONTAINERS` is `false` then `rabbitmq_container` and `redis_container` below will always be `None` so we can remove this condition.

Reply:
having `rabbitmq_container` is required to make mypy happy, otherwise it will complaint because rabbitmq_container can be None